content/download/fedora.adoc

```diff
@@ -30,29 +28,39 @@ to install a testing package:
 [source,bash]
 dnf --enablerepo=updates-testing install kicad
 
+As of KiCad 5.0.0, the 3D models have been moved to a separate package and can
+be installed with:
+
+[source,bash]
+dnf install kicad-packages3d
+
 == Nightly Development Builds
 
 Nightly development builds for Fedora are available via the
-link:https://copr.fedorainfracloud.org/coprs/g/kicad/kicad/[copr build
-service].
+link:https://copr.fedorainfracloud.org/coprs/g/kicad/kicad/[copr build service].
 
 This build can be installed on Fedora with:
 
-----
+[source,bash]
 dnf copr enable @kicad/kicad
 dnf install kicad
-----
+
+The kicad package includes the latest revision of all libraries except for the
+3D models from the
+link:https://github.com/KiCad/kicad-packages3D[kicad-packages3D] repository.
+These can be installed separately with:
```

Comment:
Would it be good to state the reason for it not being included by default? I think it is not obvious for the newcomer when reading this page. What do you think?

Reply:
You're probably right. I'll update my patch.